### ISSUE TITLE
Publish posts from planet LibreCores on twitter

### ISSUE THREAD

**Issue by olofk** [librecores/librecores-web#157]
It could be handy and increase visibility if we can make the librecores twitter account automatically tweet whenever there's a new post on planet LibreCores.

**Comment by wallento** [comment]
I see this ambivalent. On the one hand it is good to have more activity, on the other hand it can quickly become a spamming account. I feel we would be better off with either:

- something moderated
- a different account ("LibreCores Planet")

**Comment by olofk** [comment]
I don't think this is an issue. There are about 10 posts each month. If it would turn out to be a problem, we can just turn it off

**Comment by oleg-nenashev** [comment]
I would rather suggest a different account in longer-term. Fine for now
